WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -419,6 +436,46 @@ class PostListMainViewModel @Inject constructor(
                 false
             }
 
+    fun handleRemotePreviewClosing() {
+        updatePreviewAndDialogState(PostListRemotePreviewState.NONE, PostInfoType.PostNoInfo)
+    }
+
+    private fun updatePreviewAndDialogState(newState: PostListRemotePreviewState, postInfo: PostInfoType) {
+        /* We need only transitions, so... */
+        if (_previewState.value == newState) return
+
+        AppLog.d(AppLog.T.POSTS, "****** POSTLIST PREVIEW SM - from [${_previewState.value}] to [$newState]")
```

Comment:
I'd probably remove that log all together, not sure if you used it for debugging or if you intend to track bugs from our logs using that.

If you want to keep it, can we remove the formatting and caps here, we don't do this in other log entries.

Reply:
It remained there from a debug session; but thinking at it again if you don't mind I would prefer to leave it (changing the formatting) to trace eventual bugs in future. Changed here 02f8676ecca8bb8ae8e3b8e70a3b150214c7ce9c. Let me know if it's fine for you.